app/scripts/app.js

```diff
@@ -13,6 +13,7 @@
         })
         .state('album', {
```

Comment:
Right so this is what we call a 'fluent interface' (https://www.martinfowler.com/bliki/FluentInterface.html).
It's a particular way of creating an API that makes it easy to see how to do things. You'll find this commonly with configuration object build-ups like this.